src/frontends/onnx/frontend/src/op/pad.cpp

```diff
@@ -73,7 +74,7 @@ OutputVector pad(const Node& node) {
     Output<ngraph::Node> padding_end;
 
     if (inputs.size() == 3 && !ngraph::op::is_null(inputs[2])) {
-        values = inputs[2];
+        values = reshape::interpret_as_scalar(inputs[2]);
```

Comment:
Corresponding unit test for onnx model with Pad 1D "constant_value" input should be added.

Reply:
done ;)